.github/ISSUE_TEMPLATE/release_steps.md

```diff
@@ -19,18 +19,23 @@ This steps have to be followed always when preparing a new release.
 
 ## New stable branch creation
 
-If stable release (YYYY.XX.00) follow these sub-steps:
-
-- [ ] create a branch `YYYY.XX.xx` from master branch  (`xx` is really `xx`, example: 2018.01.xx). This is the new **stable branch**
-- [ ] Change [MapStore2-QA-Build](http://build.geosolutionsgroup.com/view/MapStore/job/MapStore/view/MapStore%20QA/job/MapStore2-QA-Build/) by updating the `branch` parameter in the build configuration page to `YYYY.XX.xx`
-- [ ] on MapStore **stable branch**
-    - [ ] Fix `pom.xml` files to make sure that no `-SNAPSHOT` **dependencies** are used anymore.
-- [ ] on MapStore **master branch**
-    - [ ] increase version of java modules. (`mvn versions:set -DnewVersion=<SNAPSHOT_VERSION> -DprocessAllModules -DgenerateBackupPoms=false`). Where `<SNAPSHOT_VERSION>` increases the major number. (e.g. `1.3-SNAPSHOT` --> `1.4-SNAPSHOT`)
-    - [ ] Manually update project pom templates to use `mapstore-services` of `<SNAPSHOT_VERSION>` to the new one. (`projects/templates/web/pom.xml`).
-    - [ ] Increment version of `package.json` on master **0.&lt;x-incremented&gt;.0** with the command `npm version minor --git-tag-version=false`
+**Only** if you are need to create a new stable major release (YYYY.XX.00), you need first to create a branch for it. Follow the following:
+
+- [ ] Run the [`Cut Release Branch`](https://github.com/geosolutions-it/MapStore2/actions/workflows/cut_major_branch.yml) workflow on github.
+      With the following Parameters:
+        - Use workflow from branch `master`
+        - MapStore branch name to use: `YYYY.XX.xx`
+        - Version of *MapFish Print*, *GeoStore* and *HTTP-Proxy* accordingly to the [MapStore release calendar](https://github.com/geosolutions-it/MapStore2/wiki/MapStore-Release-Calendars)
+        - use the default value for the other parameters
```

Comment:
![image](https://github.com/geosolutions-it/MapStore2/assets/11991428/c0b51c8a-d4b2-4e9c-ade6-451ddd6fe014)
not well formatted now

```suggestion
      With the following Parameters:
      
        - [ ] Use workflow from branch `master`
        - [ ] MapStore branch name to use: `YYYY.XX.xx`
        - [ ] Version of *MapFish Print*, *GeoStore* and *HTTP-Proxy* accordingly to the [MapStore release calendar](https://github.com/geosolutions-it/MapStore2/wiki/MapStore-Release-Calendars)
        - [ ] use the default value for the other parameters
```